packages/zk-transfer-web/src/libs/txHelpers.ts

```diff
@@ -0,0 +1,82 @@
+import BN from "bn.js";
+import { Map } from "immutable";
+import { Demo, EthAddress, BM, SignerEth } from "./types";
+import { untilTxMined } from "demo-tx";
+import { runTransforms } from "demo-transform";
+import { deployed, constructMintPipeline } from "demo-aztec-lib";
+import { RINKEBY_ADMIN_ADDRESS, RINKEBY_ADMIN_PASSWORD } from "./constants";
+
+const mintPipeline = constructMintPipeline();
+
+export const mint = async (state: any) => {
+  return await runTransforms(mintPipeline, state);
+};
+
+type makeMintOptions = { bm: BM; tradeSymbol: string; amount: number };
+export const makeMint = async (
+  demo: Demo,
+  { bm, tradeSymbol, amount }: makeMintOptions
+): Promise<void> => {
+  console.info("Beginning Mint Process:");
+  const { signerEth } = await demo.keys.wallet.prepareSignerEth({
+    address: RINKEBY_ADMIN_ADDRESS,
+    password: RINKEBY_ADMIN_PASSWORD
+  });
+
+  const params = Map({
+    bm,
+    tradeSymbol,
+    mintFromZero: false,
+    chainId: demo.chainId,
+    deployerEth: signerEth,
+    minteeAmount: new BN(amount),
+    minteeAddress: demo.thisAddress,
+    minteePublicKey: demo.thisPublicKey,
+    deployerAddress: RINKEBY_ADMIN_ADDRESS,
+    deployerPassword: RINKEBY_ADMIN_PASSWORD,
+    deployed: deployedFunc({ bm, signerEth }),
+    minedTx: minedTxFunc({ demo, fromAddress: RINKEBY_ADMIN_ADDRESS })
+  });
+
+  await mint(params);
+};
+
+type deployFuncProp = {
+  bm: BM;
+  signerEth: SignerEth;
+};
+
+const deployedFunc = ({ bm, signerEth }: deployFuncProp) => {
+  return async (contractName: any, options: any) => {
+    return await deployed({
+      bm,
+      options,
+      signerEth,
+      contractName
+    });
+  };
+};
+
+type minedTxProps = {
+  demo: Demo;
+  fromAddress: EthAddress;
+};
+
+const minedTxFunc = ({ demo, fromAddress }: minedTxProps) => {
+  // NOTE: We rely on `deployed` being called above to prepare the
+  // associated signer.
+  return async (method: any, argList: any, options: any): Promise<any> => {
+    const signerEth = demo.keys.wallet.signersMap[fromAddress];
+    const _options = Map({ from: fromAddress, gas: "6700000" })
```

Comment:
"6700000" rather than being hard-coded should be taken from
`demo.config['GAS_LIMIT']`